espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Router.java

```diff
@@ -178,6 +179,8 @@ public BaseResponse route(String uri, Method method, Map<String, String> params,
             return new ErrorResponse(NanoHTTPD.Response.Status.NOT_FOUND, e.getMessage());
         } catch (NotYetImplementedException e) {
             return new ErrorResponse(NanoHTTPD.Response.Status.NOT_IMPLEMENTED, e.getMessage());
+        } catch (StaleElementException e) {
+            return new AppiumResponse<>(AppiumStatus.STALE_ELEMENT_REFERENCE, e.getMessage());
```

Comment:
Not sure if we have to or not. I put it in there because it's part of the spec. @jlipps do we need to keep the MJSONWP error messages or can we use our own?